vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/rewards/legacy_rewards_migration_transaction_history_json_reader_util.cc

```diff
@@ -86,7 +86,7 @@ absl::optional<TransactionList> ParseTransactionHistory(
   const base::Value::Dict* transaction_history =
       value.FindDict(kTransactionHistoryKey);
   if (!transaction_history) {
-    return absl::nullopt;
+    return TransactionList{};
```

Comment:
We should return an empty transaction list if `kTransactionHistoryKey` node does not exist